client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -106,23 +122,16 @@ const LoggedInForm = React.createClass( {
 		return this.props.jetpackConnectAuthorize && this.props.jetpackConnectAuthorize.isAuthorizing;
 	},
 
-	dismissNotice() {
-		debug( 'Notice dissmissed' );
-	},
-
 	renderNotices() {
-		const { authorizeSuccess, authorizeError } = this.props.jetpackConnectAuthorize;
-		if ( authorizeSuccess ) {
-			return <JetpackConnectNotices noticeType="authorizeSuccess" onDismissClick={ this.dismissNotice } />;
-		}
+		const { authorizeError } = this.props.jetpackConnectAuthorize;
 		if ( authorizeError ) {
-			return <JetpackConnectNotices noticeType="authorizeError" onDismissClick={ this.dismissNotice } />;
+			return <JetpackConnectNotices noticeType="authorizeError" />;
 		}
 		return null;
 	},
 
 	renderFormControls() {
-		const { queryObject, isAuthorizing, autoAuthorize, authorizeSuccess } = this.props.jetpackConnectAuthorize;
+		const { queryObject, isAuthorizing, autoAuthorize, authorizeSuccess, plansURL, siteReceived } = this.props.jetpackConnectAuthorize;
```

Comment:
as I commented in slack, I think `PlansURL` would be better fitted to be a constant  in this file instead of receiving it from jetpack. This way we would be much more agile if we need to point that redirection to somewhere else.